### PR TITLE
1.7.8.2 release note

### PR DESCRIPTION
**Pull request by fabienvallon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Rlease note for the 1.7.8.2 release
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+- --
```
Comment (sowbiba) [inline news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md]:
```suggestion
---
```

File: news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- 1.7
+- 1.7.8.x
+- version
+- patch
+- releases
+
+- --
+
+*Intro*
+
```
Comment (sowbiba) [inline news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md]:
```suggestion
- --

*Intro*

```

File: news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+We suggest upgrading your shop quickly in order to benefit from these fixes. Of course, don’t forget to backup before.
+
+Reminder: the 1-Click Upgrade module’s latest version is v4.13.0, don’t forget to upgrade it.
+
+*Content*
+
+---
+
+## ## Main fixes (change content)
```
Comment (sowbiba) [inline news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md]:
```suggestion
## Main fixes
```

File: news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+Below are listed some of the one regression issue we fixed in this version :
```
Comment (sowbiba) [inline news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md]:
```suggestion
Below is the one regression issue we fixed in this version :
```

File: news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+- Fix wishlist button disappearance on the category page while using any filter in faceted search.([[#](https://github.com/PrestaShop/PrestaShop/issues/15834)[26327](https://github.com/PrestaShop/PrestaShop/issues/26327)]([https://github.com/PrestaShop/PrestaShop/issues/26327](https://github.com/PrestaShop/PrestaShop/issues/26327)))
```
Comment (sowbiba) [inline news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md]:
```suggestion
- Fix wishlist button disappearance on the category page while using any filter in faceted search.([#26327](https://github.com/PrestaShop/PrestaShop/issues/26327))
```

File: news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+Read the [[full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.6.2)](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.8.2).
```
Comment (sowbiba) [inline news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md]:
```suggestion
Read the ([full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.8.2)).
```

File: news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+## ## Security fix (change content)
```
Comment (sowbiba) [inline news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md]:
```suggestion
## Security fix
```

File: news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+Contributors to this patch version, from both the Core team and the community at large:
+
+Alexandre Bru, Mathieu Ferment, Pierre Rambaud, Valentin Szczupak.
+
+Thank you!
+
+[Download PrestaShop 1.7.8.2 now!](https://www.prestashop.com/en/download){: .btn .btn-lg .btn-success}
+
+{: .text-center}
+
```
Comment (sowbiba) [inline news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md:59]:
```suggestion
{: .text-center}

```

**Review by sowbiba — DISMISSED** [review]

**Review by matks — APPROVED** [review]
It's OK for me.

Just modify the publish hour before merge :)

**Comment by MatShir** [comment]
The release not says it fixes one issue, but the changelog says that it fixes 2 ? Which one should we trust ?

**Review by MatShir — CHANGES_REQUESTED** [review]
Should we mentioned the second bug?

File: news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+Reminder: the 1-Click Upgrade module’s latest version is v4.13.0, don’t forget to upgrade it.
+
+
+## Main fix
+
+Below is the one regression issue we fixed in this version :
+
+- Fix wishlist button disappearance on the category page while using any filter in faceted search ([#26327](https://github.com/PrestaShop/PrestaShop/issues/26327))
+
+Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.8.2).
```
Comment (atomiix) [inline news/_posts/releases/2021-12-07-prestashop-1-7-8-2-maintenance-release.md]: I think we usually use the changelog uploaded on the .com, but maybe you're right to use the one on github now that we try to separate things.
```suggestion
Read the [full changelog here](https://download.prestashop.com/download/releases/changelog_1.7.8.2.txt).
```

**Review by atomiix — APPROVED** [review]
1 comment but approving anyway as it's optionnal

**Review by MatShir — APPROVED** [review]

**Comment by sowbiba** [comment]
Thanks @fabienvallon

**Pull request merged**